bundles/org.openhab.binding.shelly/src/test/java/org/openhab/binding/shelly/internal/api/ShellyDeviceProfileTest.java

```diff
@@ -119,7 +119,6 @@ private static Stream<Arguments> provideTestCasesForApiDetermination() {
                 Arguments.of(THING_TYPE_SHELLYPRO4PM, true, false), //
                 Arguments.of(THING_TYPE_SHELLYPLUSDIMMERUS, true, false), //
                 Arguments.of(THING_TYPE_SHELLYPLUSDIMMER10V, true, false), //
-                Arguments.of(THING_TYPE_SHELLYPLUSHTG3, true, false), //
```

Comment:
See https://github.com/openhab/openhab-addons/pull/18943#pullrequestreview-3014350077

Please keep this test for `THING_TYPE_SHELLYPLUSHT`.